src/solver/conv_hip_implicit_gemm_bwd_data_xdlops.cpp

```diff
@@ -287,7 +287,7 @@ bool PerformanceConfigHipImplicitGemmBwdXdlops::IsValid(const ProblemDescription
 bool PerformanceConfigHipImplicitGemmBwdXdlops::operator==(
     const PerformanceConfigHipImplicitGemmBwdXdlops& other) const
 {
-    return this->index == other.index;
+    return this->index == other.index && this->total_size == other.total_size;
```

Comment:
_Topic continued from https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1874#discussion_r1023495820_

[Important] @iq136boy 
> We might need to add the comparison of total_size too. 

Why?

Reply:
> _Topic continued from [#1874 (comment)](https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1874#discussion_r1023495820)_
> 
> [Important] @iq136boy
> 
> > We might need to add the comparison of total_size too.
> 
> Why?

@atamazov This is a bit confusing here. Now I think only to compare the kernel_id since that will guarantee to get the kernel instance we want.